cpp/cgi_progs/translator.cc

```diff
@@ -720,8 +720,8 @@ void MailMyTranslations(DbConnection &db_connection, const IniFile &ini_file, co
     if (recipient.empty())
         return;
 
-    if (unlikely(not SendEmail("no-reply@ub.uni-tuebingen.de", recipient, "Your IxTheo Translations",
-                               mail_content.str(), EmailSender::DO_NOT_SET_PRIORITY, EmailSender::HTML)))
+    if (unlikely(not EmailSender::SimpleSendEmail("no-reply@ub.uni-tuebingen.de", { recipient }, "Your IxTheo Translations",
```

Comment:
Do we really need sender, priority and format in the simplest version?

Reply:
Yes, we do as you will see when you continue reading the rest of the PR.  I still think that "Simple" instead of "Simpler" is the better prefix.